systems/robotInterfaces/Side.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SYSTEMS_CONTROLLERS_SIDE_H_
+#define SYSTEMS_CONTROLLERS_SIDE_H_
+
+#include <array>
+#include <string>
+
+// adapted from https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Type_Safe_Enum
+class Side
```

Comment:
Any suggestions? We used RobotSide at IHMC, but I always found there was nothing robot-specific about left or right. Maybe just LeftRight?